src/ImageProcessor.Web/HttpModules/ImageProcessingModule.cs

```diff
@@ -214,12 +214,32 @@ public static void SetHeaders(HttpContext context, string responseType, string[]
 
             cache.SetExpires(DateTime.Now.ToUniversalTime().AddDays(maxDays));
             cache.SetMaxAge(new TimeSpan(maxDays, 0, 0, 0));
-            cache.SetRevalidation(HttpCacheRevalidation.AllCaches);
+            //cache.SetRevalidation(HttpCacheRevalidation.AllCaches);
```

Comment:
hmm, don't remember commenting out that - must have just been testing it out. Sorry about leaving it in...